src/frontends/onnx/tests/onnx_import.in.cpp

```diff
@@ -910,6 +910,35 @@ OPENVINO_TEST(${BACKEND_NAME}, onnx_model_reduce_log_sum) {
     test_case.run();
 }
 
+OPENVINO_TEST(${BACKEND_NAME}, onnx_model_reduce_log_sum_18) {
+    auto model = convert_model("reduce_log_sum_18.onnx");
+
+    // input data shape (1, 1, 4, 4)
+    Inputs inputs{
+        ov::test::NDArray<float, 4>({{{{1, 1, 1, 1}, {1, 1, 1, 1}, {1, 1, 1, 1}, {1, 1, 1, 1}}}}).get_vector()};
+
+    // output data shape (1,)
+    auto expected_output = ov::test::NDArray<float, 4>({{{{2.77258872f}}}}).get_vector();
+
+    auto test_case = ov::test::TestCase(model, s_device);
+    test_case.add_multiple_inputs(inputs);
+    test_case.add_expected_output(expected_output);
+    test_case.run();
+}
+
+OPENVINO_TEST(${BACKEND_NAME}, onnx_model_reduce_log_sum_18_axes_as_input) {
+    auto model = convert_model("reduce_log_sum_18_axes_as_input.onnx");
+    auto test_case = ov::test::TestCase(model, s_device);
+
+    test_case.add_input<float>(Shape{1, 1, 4, 4}, {2, 1, 4, 2, 3, 1, 3, 2, 4, 2, 4, 2, 2, 2, 1, 4});
+    test_case.add_input<int64_t>({3});
+
+    test_case.add_expected_output(Shape{1, 1, 4, 1},
+                                  std::vector<float>{2.19722458, 2.19722458, 2.48490665, 2.19722458});
```

Comment:
Message from the [windows CI log](https://github.com/openvinotoolkit/openvino/actions/runs/8379345729/job/22946136029?pr=23508#step:15:6469):

`openvino_build\src\frontends\onnx\tests\onnx_import_INTERPRETER.cpp(937): warning C4305: 'initializing': truncation from 'double' to '_Ty'
`

The following change should fix the issue:

```suggestion
                                  std::vector<float>{2.19722458f, 2.19722458f, 2.48490665f, 2.19722458f});
```

Reply:
Thank you